src/Workspaces/Core/Portable/Simplification/Simplifier.cs

```diff
@@ -37,12 +37,6 @@ namespace Microsoft.CodeAnalysis.Simplification
     /// </summary>
     public static partial class Simplifier
     {
-        /// <summary>
-        /// The annotation the reducer uses to identify sub trees to be reduced.
-        /// The Expand operations add this annotation to nodes so that the Reduce operations later find them.
-        /// </summary>
-        public static SyntaxAnnotation Annotation { get; } = new SyntaxAnnotation();
```

Comment:
😕 This is a public API. Why are we duplicating it?

Reply:
WIP.  :)